README.md

```diff
@@ -130,8 +130,26 @@ pip install git+ssh://git@github.com/equinor/fmu-dataio@0.1.1
 
 Local development and testing:
 
+Make your own fork of fmu-dataio and then clone it locally on unix.
```

Comment:
Perhaps just "create fork" (doesn't have to be on Unix, technically, I think.)

Also, should we specify general guidelines here? Aka "Do local development in feature branches on personal forks" or similar?

@jcrivenaes inputs?